Library/Formula/xorg.rb

```diff
@@ -0,0 +1,470 @@
+        system "make", "install"
+      end
+    end
+
+#   ENV.append_path "PATH",                "#{prefix}/bin"
+    ENV.append_path "PKG_CONFIG_PATH",     "#{prefix}/lib/pkgconfig"   # xcb-proto is there 
+    ENV.append_path "PKG_CONFIG_PATH",     "#{prefix}/share/pkgconfig" # xorg-macros is there
+#   ENV.append_path "LIBRARY_PATH",        "#{prefix}/lib"
+#   ENV.append_path "C_INCLIDE_PATH",      "#{prefix}/include"
+#   ENV.append_path "CPLUS_INCLUDE_PATH",  "#{prefix}/include"
+
+    mkdir_p lib
+    ln_s lib, "#{prefix}/lib64"
+
+# libXdmcp
+    resource("libXdmcp").stage do
+      system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{prefix}/etc", "--localstatedir=#{prefix}/var", "--disable-static"
+      system "make"
+      system "make", "install"
+    end
+
+
+# libXau
+    resource("libXau").stage do
+      system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{prefix}/etc", "--localstatedir=#{prefix}/var", "--disable-static" 
+      system "make"
+      system "make", "check"
+      system "make", "install"
+    end
+
+# xcb-proto
+    resource("xcb-proto").stage do
+      system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{prefix}/etc", "--localstatedir=#{prefix}/var", "--disable-static"
+      system "make", "check"
+      system "make", "install"
+    end
+
+### libpthread-stubs
+### see inreplace below 
+##    resource("libpthread-stubs").stage do
+##      system "./configure", "--prefix=#{prefix}"
+##      system "make"
+##      system "make", "install"
+##    end
+
+# libxcb
+    resource("libxcb").stage do
+      inreplace "configure", /pthread-stubs/, ""
+      system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{prefix}/etc", "--localstatedir=#{prefix}/var", "--disable-static",
+                            "--enable-xinput" # , "--without-doxygen"
+
+      system "make"
+      system "make", "install"
+    end
+
+ 
+    res = %w[xtrans libX11 libXext libFS libICE libSM libXScrnSaver libXt libXmu libXpm libXaw libXfixes libXcomposite libXrender libXcursor libXdamage libfontenc libXfont libXft libXi libXinerama libXrandr libXres libXtst libXv libXvMC libXxf86dga libXxf86vm libdmx libpciaccess libxkbfile libxshmfence]
+
+    res.each do |r|
+      resource(r).stage do
+
+      case r 
+      when "libXfont"
+        system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{prefix}/etc", "--localstatedir=#{prefix}/var", "--disable-static", "--disable-devel-docs"
+      when "libXt"
+        system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{prefix}/etc", "--localstatedir=#{prefix}/var", "--disable-static", "--with-appdefaultdir=#{prefix}/X11/app-defaults"
+      else
+        system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{prefix}/etc", "--localstatedir=#{prefix}/var", "--disable-static"
+      end
+
+        system "make"
+        system "make", "install"
+        system "ldconfig"
```

Comment:
`ldconfig` shouldn't be necessary. Linuxbrew uses `RPATH` rather than `ld.so.conf`.